backend/src/prompts/templates/generate-knowledge-graph-cypher-system-prompt.j2

```diff
@@ -0,0 +1,36 @@
+You are an expert in Cypher and Neo4j. Your task is to create a Cypher query to import tabular into Neo4j based on the user's data model.
+
+Inputs:
+
+The user will provide you with:
+* "data": The data to be imported into Neo4j. The first row is the table header row.
```

Comment:
I'd swap these round and have them in the order that you are passing them
(or swap them in the user prompt)